superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx

```diff
@@ -39,7 +40,19 @@ interface DatabaseModalProps {
   database?: DatabaseObject | null; // If included, will go into edit mode
 }
 
+// todo: define common type fully in types file
+interface RootState {
+  common: {
+    conf: {
+      SQLALCHEMY_DOCS_URL: string;
+    };
+  };
+  messageToast: Array<Object>;
+}
+
 const DEFAULT_TAB_KEY = '1';
+const DEFAULT_SQLALCHEMY_DOCS_URL =
+  'https://docs.sqlalchemy.org/en/rel_1_2/core/engines.html#';
```

Comment:
do you want to put this in the default config file instead?

Reply:
Actually I see you also have it there.. seems like unless someone overwrites it with an empty string in their own config, it should always exist and we shouldn't need a default here as well?

Reply:
The default i'm using since the unit test aren't playing nice, i couldn't figure out how to get the `wrappingComponent must render its children!`